rpcserver.go

```diff
@@ -2841,8 +2841,7 @@ func SignRawTransaction(w *Wallet, chainSvr *chain.Client, icmd btcjson.Cmd) (in
 
 		// Either it was already signed or we just signed it.
 		// Find out if it is completely satisfied or still needs more.
-		flags := txscript.ScriptBip16 | txscript.ScriptVerifyDERSignatures |
-			txscript.ScriptStrictMultiSig
+		flags := txscript.StandardVerifyFlags
```

Comment:
const (or just pass in in directly, it was a var before so they could be modified for bip16)